InMobi/src/main/java/com/mopub/mobileads/InMobiSDKAdapterConfiguration.kt

```diff
@@ -17,15 +21,29 @@ class InMobiSDKAdapterConfiguration : BaseAdapterConfiguration() {
     }
 
     override fun getMoPubNetworkName(): String {
-        return "inmobi"
+        return BuildConfig.NETWORK_NAME
     }
 
     override fun getNetworkSdkVersion(): String {
         return InMobiSdk.getVersion()
     }
 
     override fun initializeNetwork(context: Context, configuration: Map<String, String>?, onNetworkInitializationFinishedListener: OnNetworkInitializationFinishedListener) {
-        onNetworkInitializationFinishedListener.onNetworkInitializationFinished(InMobiSDKAdapterConfiguration::class.java,
-                MoPubErrorCode.ADAPTER_INITIALIZATION_SUCCESS)
+        if (null != configuration) {
+            val accountId = if (configuration.containsKey(ACCOUNT_ID)) configuration[ACCOUNT_ID] else null
+            if (!accountId.isNullOrEmpty()) {
+                InMobiSdk.init(context, accountId, InMobiGDPR.gdprConsentDictionary) { error ->
+                    if (null == error) {
+                        onNetworkInitializationFinishedListener.onNetworkInitializationFinished(InMobiSDKAdapterConfiguration::class.java, MoPubErrorCode.ADAPTER_INITIALIZATION_SUCCESS)
+                    } else {
+                        onNetworkInitializationFinishedListener.onNetworkInitializationFinished(InMobiSDKAdapterConfiguration::class.java, MoPubErrorCode.ADAPTER_CONFIGURATION_ERROR)
+                    }
+                }
```

Comment:
You can call `return` from here and then you would not need to have two `else`'s which basically are doing the same thing.